modules/navigation/src/main/res/layout/layout_lanes_view_stub.xml

```diff
@@ -28,27 +23,24 @@
 
     <data>
 
-        <import type="android.view.View" />
+        <import type="android.view.View"/>
 
         <variable
-            name="navigationFragmentViewModel"
-            type="com.sygic.maps.module.navigation.viewmodel.NavigationFragmentViewModel" />
+            name="lanesViewModel"
+            type="com.sygic.maps.uikit.viewmodels.navigation.lanes.LanesViewModel" />
 
         <variable
-            name="signpostViewModel"
-            type="com.sygic.maps.uikit.viewmodels.navigation.signpost.SimplifiedSignpostViewModel" />
+            name="navigationFragmentViewModel"
+            type="com.sygic.maps.module.navigation.viewmodel.NavigationFragmentViewModel" />
 
     </data>
 
-    <com.sygic.maps.uikit.views.navigation.signpost.SimplifiedSignpostView
-        android:layout_width="344dp"
+    <com.sygic.maps.uikit.views.navigation.lanes.SimpleLanesView
+        android:layout_width="@dimen/lanesViewWidth"
         android:layout_height="wrap_content"
-        android:visibility="@{safeUnbox(navigationFragmentViewModel.signpostEnabled) ? View.VISIBLE : View.GONE}"
-        app:distance="@{signpostViewModel.distance}"
-        app:instructionText="@{signpostViewModel.instructionText}"
-        app:primaryDirection="@{signpostViewModel.primaryDirection}"
-        app:secondaryDirection="@{signpostViewModel.secondaryDirection}"
-        app:secondaryDirectionContainerVisible="@{safeUnbox(signpostViewModel.secondaryDirectionContainerVisible)}"
-        app:secondaryDirectionText="@{signpostViewModel.secondaryDirectionText}" />
+        android:layout_marginTop="8dp"
+        app:lanes="@{lanesViewModel.lanes}"
+        android:visibility="@{safeUnbox(navigationFragmentViewModel.lanesViewEnabled) &amp;&amp; safeUnbox(lanesViewModel.enabled) ? View.VISIBLE : View.GONE}">
 
+    </com.sygic.maps.uikit.views.navigation.lanes.SimpleLanesView>
```

Comment:
stačí to jen zavřít klasicky v deklaraci pomocí `/>`, když to nebude mít žádné děti, či?